common/src/main/java/tds/common/logging/EventLogger.java

```diff
@@ -19,6 +19,12 @@
  * Centralized event logger.
  */
 public class EventLogger {
+
+    public enum LogEvent {
+        APP_STARTUP,
```

Comment:
I almost wish we would have created a separate project that had all this logging stuff that you could include rather than putting into individual projects (common, and ss_shared_code).  For example, we'd have a TDS_Logging Github project that would house all this logging stuff.

@mtrupkin @jtreuting , it is probably too late in the project to do what I suggest below.  Just a suggestion for phase 2.